### PR TITLE
fix(i18n): translated some fields for `addon-editor`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/i18n/languages/russian/addon-editor.ts [hunk 1]
```diff
@@ -50,9 +50,9 @@ export const TUI_RUSSIAN_LANGUAGE_ADDON_EDITOR: LanguageEditor = {
     ],
     editorCodeOptions: ['Код', 'Код в блоке'],
     editorFontOptions: {
-        small: 'Small',
+        small: 'Маленький',
```
Comment (nsbarsukov) [inline projects/i18n/languages/russian/addon-editor.ts:53]: I think "Мелкий" is more appropriate =)

**Proofs:**
1. Open chrome://settings/appearance
2. Click "Font-size" button
chrome

Comment (nsbarsukov) [reply projects/i18n/languages/russian/addon-editor.ts:53]: Also, the same with `Большой` => `Крупный`

Comment (nsbarsukov) [reply projects/i18n/languages/russian/addon-editor.ts:53]: But i mark this PR as `nit`)

**Review by nsbarsukov — APPROVED** [review]

File: projects/i18n/languages/russian/addon-editor.ts (hunk 1 above)
Comment (splincode) [reply projects/i18n/languages/russian/addon-editor.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**